content/compose/reference/_index.md

```diff
@@ -126,12 +113,12 @@ relative to the current working directory.
 
```

Comment:
unrelated to the changes here, but we could probably use an example here for reading the compose file from stdin with a heredoc